Makefile

```diff
@@ -136,6 +143,13 @@ snap:
 	$(SNAPCRAFT)
 .PHONY: snap
 
+# TICS expects coverage info to be in ./coverage/.coverage
+tics-analysis: depends-ci coverage
+	sudo apt install pylint
+	mkdir -p coverage
+	mv .coverage ./coverage/.coverage
```

Comment:
We may want to consider copying these instead. I know this is CI specific, but in case we may want to run this locally. I also think `clean` should be updated to remove the copied files in case they exist. Add to `gitignore` as well.

Reply:
Sure.  This is a good reminder to tweak the name a bit, too.  It doesn't really  *run* TiCS analysis, it just prepares the workspace for it.  So I'll name it `prepare-tics-analysis` instead.